scripts/cloudfront_routing.js

```
@@ -0,0 +1,53 @@
+/**
+ * This script follows ES5.1 rules for compatibility.
+ * - Avoids ES6+ syntax (e.g., `let`, `const`, arrow functions).
+ */
+'use strict';
+function handler(event) {
+    var request = event.request;
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)